README.md

```diff
@@ -43,6 +51,129 @@ class MyParameterizedTest {
 }
 ```
 
+### Android tests
+
+#### Setting up the dependency
+```gradle
+repositories {
+    maven { url 'https://dl.bintray.com/qameta/maven' }
+}
+
+dependencies {
+    androidTestImplementation "io.qameta.allure:allure-kotlin-model:$LATEST_VERSION"
+    androidTestImplementation "io.qameta.allure:allure-kotlin-commons:$LATEST_VERSION"
+    androidTestImplementation "io.qameta.allure:allure-kotlin-junit4:$LATEST_VERSION"
+    androidTestImplementation "io.qameta.allure:allure-kotlin-android:$LATEST_VERSION@aar"
+}
+
+android {
```

Comment:
I feel like this shouldn't be part of the `Setting up the dependency` section. There are 2 ways of attaching the listener
- using `AllureAndroidJUnit4` over class - works for both robolectric & on-device tests
- using `testInstrumentationRunner` - works only for `on-device` tests

I fell like moving this bit to `On-device instrumentation tests` would be more fitting.